src/main/java/com/university/university/service/AtUniversityServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.university.university.service;
+
+import com.university.university.model.Course;
+import com.university.university.repository.CourseRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class AtUniversityServiceImpl implements AtUniversityService{
+
+    @Autowired
+    private CourseRepository courseRepository;
+
+    @Override
+    public List<Course> getCourses() {
+        List<Course> courses = new ArrayList<>();
+
+        courses = courseRepository.findAll();
+        return courses;
```

Comment:
Courses variable is not needed, we can return the list directly in Return statement.